IS7024/Pages/HollywoodBowl.cshtml

```diff
@@ -4,10 +4,15 @@
     ViewData["Title"] = "HollywoodBowl";
     var events = (List<QuickTypeBowl.Event>)ViewData["events"];
     var sun = ViewData["sun"];
+    string eventlocation = (string)ViewData["eventlocation"];
+
 }
 
 <div class="text-center">
     <h1 class="display-4">Upcoming Events at the Hollywood Bowl</h1>
+    @{
+        <h4>@eventlocation</h4>
+    }
```

Comment:
This was added so that the city and state of the venue is pulled from the JSON data for that venue instead of written in.